bluebrain/repo-bluebrain/packages/py-atlalign/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAtlalign(PythonPackage):
+    """Blue Brain multi-modal registration and alignment toolbox."""
+
+    homepage = "https://pypi.org/project/atlalign/"
+    pypi = "atlalign/atlalign-0.6.1.tar.gz"
+
+    version('0.6.1', sha256='8e3532987ab284211c3408a47855fb0deb439b93d58d92dce44e95a530846b0a')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+
+    depends_on('py-antspyx@0.3.1', type=('build', 'run'))
```

Comment:
Hi @EmilieDel, I opened this PR to update `gene-expression-volume` and its dependencies in spack, including `atlalign 0.6.1`.
The build is successful now, but I needed to set the `py-antspyx` dependency to the version 0.3.1.

I see that:
- `atlalign` depends on `antspyx==0.2.7` in https://github.com/BlueBrain/atlas-alignment/blob/54b7c2061d94f3f7e8ee27b98f64a024558fa43b/setup.py#L20
- but the source and wheel packages in https://pypi.org/project/atlalign/0.6.1/#files depend on `antspyx==0.3.1`

I wonder if you can help to clarify what is the correct dependency?

As a side note, antspyx causes a lot of warnings to be printed when running gene-expression-volume,  but I'm not sure if we can or want to do something about it.
```
/gpfs/bbp.cscs.ch/ssd/apps/bsd/pulls/1527/stage_applications/install_gcc-9.4.0-skylake/py-antspyx-0.3.1-nofauo/lib/python3.9/site-packages/ants/core/ants_image.py:611: DeprecationWarning:

inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
```

Reply:
Hello @GianlucaFicarelli, sorry about that! We can go with `antspy==0.3.1`. I am going to open a PR on `atlalign` to correct this. 

Thanks for the side note, I am going to have a look at it.